sparta/sparta/resources/Queue.hpp

```diff
@@ -329,28 +329,28 @@ namespace sparta
             /// Get the accurate logical index of this entry in the queue.
             uint32_t getIndex() const
             {
-                sparta_assert(unique_id_ >= attatched_queue_->numOngoingInvalidations_(),
+                sparta_assert(unique_id_ >= attached_queue->numOngoingInvalidations_(),
                             "Cannot get index. This QueueIterator does not represent a valid entry in the Queue");
-                sparta_assert(attatched_queue_ != nullptr,
+                sparta_assert(attached_queue != nullptr,
                             "Cannot get index. No Queue is attatched with this QueueEntree");
-                sparta_assert(physical_idx_ < attatched_queue_->vector_size_, "Not a valid Queue Iterator" );
-                return attatched_queue_->convertPhysicalIndex_(physical_idx_);
+                sparta_assert(physical_idx_ < attached_queue->vector_size_, "Not a valid Queue Iterator" );
+                return attached_queue->convertPhysicalIndex_(physical_idx_);
             }
 
         private:
 
-            QueuePointerType attatched_queue_;
+            QueuePointerType attached_queue;
```

Comment:
Why did you drop the trailing `_`?

Reply:
Sorry, find and replace error. Adding it back.